build.go

```diff
@@ -1,23 +1,25 @@
 package thing
 
 import (
-	"errors"
 	"fmt"
+	"log"
 	"os/exec"
 	"strings"
 )
 
 type Runner struct {
-	test    bool
-	History []string
-	Output  string
+	test      bool
+	History   []string
+	Output    string
+	updated   bool
```

Comment:
Good, and the default value will be `false`, which is correct, isn't it? So we don't need to worry about setting this when we create Runners.

Reply:
By the way, it's conventional, although it doesn't really matter, to list your _exported_ fields before your unexported ones.

Reply:
"Good, and the default value will be false, which is correct, isn't it? So we don't need to worry about setting this when we create Runners." 

Correct, the assumption is that yum is not updated and only needs to run once. 

"By the way, it's conventional, although it doesn't really matter, to list your exported fields before your unexported ones."

Okay, I moved the exported fields to the top.